### DISCUSSION TITLE
simbot结合Springboot启动时输入短信的验证码后报错

### DISCUSSION THREAD

**Discussion by TheBreezeOverflows** [simple-robot/Discussions#118] [Q&A]
### 涉及的编程语言

Java

### 涉及的组件库

mirai (QQ)

### 疑问描述

不管我用的是Springboot3.x还是Spring boot2.x相关的版本启动时总会抛出这个报错信息:
```
net.mamoe.mirai.internal.utils.crypto.DecryptionFailedException: null Caused by: 
java.lang.IllegalStateException: Bot login failed. cause: null
```
QQ也试了好几个。

**Comment by ForteScarlet** [comment]
- 参考 https://github.com/mamoe/mirai/issues/2570 ，并可以尝试更新mirai核心 `mirai-core-jvm` 版本到更新的版本（参考 [releases](https://github.com/mamoe/mirai/releases) ）

如果与无法登录无关，将异常等信息反馈到 [issues](https://github.com/simple-robot/simpler-robot/issues/new?assignees=ForteScarlet&labels=bug%3F&template=bug-report.yml)
